Dockerfile

```diff
@@ -49,29 +50,40 @@ RUN curl -L http://nginx.org/download/nginx-${NGINX_VERSION}.tar.gz > /tmp/nginx
     cd /tmp && \
     tar zxf nginx-${NGINX_VERSION}.tar.gz
 
+
 # Fetch and unarchive ngx_small_light module
 RUN curl -L https://github.com/cubicdaiya/ngx_small_light/archive/v${NGX_SMALL_LIGHT_VERSION}.tar.gz > /tmp/ngx_small_light-${NGX_SMALL_LIGHT_VERSION}.tar.gz && \
     cd /tmp && \
     tar zxf ngx_small_light-${NGX_SMALL_LIGHT_VERSION}.tar.gz && \
     cd /tmp/ngx_small_light-${NGX_SMALL_LIGHT_VERSION} && \
     ./setup
 
+
+# Prepare ngx_mruby
+RUN cd /tmp && \
+    git clone https://github.com/matsumotory/ngx_mruby && \
+    cd ngx_mruby && \
+    ./configure --with-ngx-src-root=/tmp/nginx-${NGINX_VERSION} && \
+    make build_mruby && \
+    make generate_gems_config
```

Comment:
Please remove `ngx_mruby` directory after installation to reduce total image size as possible.